lib/Progress/tests/interactor.js

```diff
@@ -0,0 +1,12 @@
+import {
+  interactor,
+  property,
+  text,
+} from '@bigtest/interactor';
+
+export default interactor(class ProgressInteractor {
+  static defaultScope = '[data-test-progress-bar]';
+
+  currentProgressStyles = property('div[class*=progressCurrent---', 'style');
```

Comment:
```suggestion
  currentProgressStyles = property('[class*=progressCurrent---', 'style');
```
Can we proceed without divs?